proto/ibc/applications/transfer/v1/query.proto

```diff
@@ -84,3 +89,14 @@ message QueryDenomHashResponse {
   // hash (in hex format) of the denomination trace information.
   string hash = 1;
 }
+
+// QueryEscrowAddressRequest is the request type for the EscrowAddress RPC method.
+message QueryEscrowAddressRequest {
+  string port_id    = 1;
+  string channel_id = 2;
+}
+
+// QueryEscrowAddressResponse is the response type of the EscrowAddress RPC method.
+message QueryEscrowAddressResponse {
+  string escrow_address = 1;
+}
```

Comment:
Could we add some protodoc comments for the message fields? Just basic standard ones we have in other places. 
Something like:

```
// unique port identifier
string port_id = 1;
// unique channel identifier
string channel_id = 2;

---

// the escrow account address
string escrow_address = 1;
```

Reply:
makes sense will add some comments!